vhdbuilder/packer/pre-install-dependencies.sh

```diff
@@ -80,6 +80,10 @@ capture_benchmark "enable_modified_log_rotate_service"
 systemctlEnableAndStart sync-container-logs.service || exit 1
 capture_benchmark "sync_container_logs"
 
+# enable and start nbcparser.service
+systemctl enable nbcparser.target
```

Comment:
should we just be consistent and use `systemctlEnableAndStart` like we do for other units? or did we decide that we only wanted to enable here?

Reply:
we just want to enable, the new service will automatically be started once cloud-init is complete